create-presigned-download-url/src/main/java/no/unit/nva/download/publication/file/RequestUtil.java

```diff
@@ -0,0 +1,93 @@
+package no.unit.nva.download.publication.file;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import no.unit.nva.download.publication.file.publication.exception.InputException;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.handlers.RequestInfo;
+import org.apache.http.HttpHeaders;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public final class RequestUtil {
```

Comment:
@axthosarouris Is this something we should add to nva-commons?

Reply:
@janbj it seems so

Reply:
I think it would be a great addition :)